service/src/test/java/uk/gov/hmcts/reform/fpl/service/CaseValidatorServiceTest.java

```diff
@@ -270,22 +234,53 @@ void shouldNotReturnAnErrorWhenGroundsAndGroundsForEPOAreCompleted() {
                 .build());
     }
 
-    private List<Element<Applicant>> initApplicants() {
+    private List<Element<Respondent>> initRespondents() {
+        return ImmutableList.of(
+            Element.<Respondent>builder()
+                .id(UUID.randomUUID())
+                .value(Respondent.builder().party(
+                    RespondentParty.builder()
+                        .firstName("Timothy")
+                        .lastName("Jones")
+                        .build())
+                    .build())
+                .build(),
+            Element.<Respondent>builder()
+                .id(UUID.randomUUID())
+                .value(Respondent.builder().party(
+                    RespondentParty.builder()
+                        .firstName("Sarah")
+                        .build())
+                    .build())
+                .build());
+    }
+
+    private List<Element<Applicant>> initApplicants(Boolean hasCompletedAddress) {
+        Address address;
```

Comment:
I would suggest that instead of having an Address variable using and Address.Builder variable and extracting the common logic of the if to before it (i.e. the addressLine2). Then when it needs to be used you can just build it.